### PR TITLE
Added the option to display or hide a header in a model

### PR DESCRIPTION
**Pull request by bupy7**

### CODE REVIEW

**Comment by bupy7** [comment]
Ongoing by https://github.com/bupy7/yii2-pages/pull/15

**Comment by bupy7** [comment]
Hello @zacksleo , @amavis442 and @maxxer ! Could you please check translation of `DISPLAY_TITLE` message at the pull-request?

File: src/messages/zh-TW/core.php [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
  * Language 'Traditional Chinese'.
- * 
+ *
  * @author zacksleo http://zacksleo.github.io
  * @since 1.2.1
  */
 return [
     'ID' => 'ID',
     'TITLE' => '文章標題',
+    'DISPLAY_TITLE' => '显示标题',
```
Comment (zacksleo) [inline src/messages/zh-TW/core.php]: should be 
```php
'DISPLAY_TITLE' => '顯示標題',
```

**Review by zacksleo — CHANGES_REQUESTED** [review]
`src/messages/zh-TW/core.php`

File: src/messages/zh-TW/core.php (hunk 1 above)
Comment (bupy7) [reply src/messages/zh-TW/core.php]: Thanks a lot!

**Comment by bupy7** [comment]
Realased https://github.com/bupy7/yii2-pages/releases/tag/v1.4.0

**Pull request closed without merging**